docs/misc/onboarding/onboarding.md

```diff
@@ -0,0 +1,226 @@
+# Onboarding
+
+*Incomplete developers guide to Onboarding in Trezor Suite*
+
+There are few different ways to trigger the onboarding process:
+- Initial run 
+  - Initial run is what we call a state when Suite is launched for the first time. It may also be triggered by clearing the app storage (flag is stored inside a reducer `suite.flags.initialRun` and saved to persistent storage). This is stored per device. If Suite detects initial run it'll automatically launch the onboarding.
+- Connecting an uninitialized device (without a seed)
+  - Suite will automatically launch the onboarding
+- Wiping a device from Suite UI and proceeding with device setup
+
+## Steps
+- [Welcome](#Welcome)
+  - [Data analytics](###Data-analytics-(only-in-initial-run))
+  - [Device security (genuinity) check](###Device-security-(genuinity)-check)
+- [Firmware](##Firmware)
+  - Note about normal and bootloader mode
+  - Firmware update or installation
+  - Device with older firmware installed
+  - Device with no firmware installed
+  - State of currently shipped devices, intermediary firmware
+- [Generating seed](##Generating-seed)
+  - [Generating new seed](###Generating-new-seed)
+  - [Recovery from mnemonic](###Recovery-from-mnemonic)
+- [Backup seed](##Backup-seed)
+- [PIN Setup](##PIN-setup)
+- [Suite Settings](##Suite-Settings) (enabled coins, custom backends, TOR)
+- [Final step](##Final-step) (device label, homescreen)
```

Comment:
yes It is basically stripped down ToC (as you can see there is not every subheading). It is pretty long document and I find having ToC quite essential (maybe later each step could even be in its own file). Yes, it needs to be synced when you change some heading or add new steps, I think that is reasonable cost for the value it provides. You could also make the same, maybe even stronger argument about syncing docs with code 😁 Or do you have some solution in mind that would allow us to still have something like ToC without additional work?

Reply:
> I think that is reasonable cost for the value it provides

Ack. I see this differently but will respect your view. Let's keep it. I'll resolve this.

> or do you have some solution in mind that would allow us to still have something like ToC without additional work?

Ideally, generating ToC is the job of the MD renderer. Personally, I try to write each individual MD file such that it doesn't need a ToC by giving it a good and general introduction eg. `this document describes this and that, but doesn't answer these or those` and then just taking care the rest of the document is well structured. I think you did this very well here that's why I just proposed to remove this ToC. .)

Reply:
> Ideally, generating ToC is the job of the MD renderer.

Eg. just like [this theme](https://github.com/squidfunk/mkdocs-material) for MkDocs does.